### ISSUE TITLE
NAV-GAC-Gestión de impagos

### ISSUE THREAD

**Issue by bpesquero** [PKFEric/webProxima#1310]
Con respecto a la gestión de impagos en Geo nos encontramos con los siguientes problemas

1.- Al enviar el email de reclamación formal de deuda y de resolución de contratos se envíe en vez de al email del cliente a un email nuestro. - Crear campo en info empresa
2.- Formato del email de resolución de contratos no es correcto, en vez de enviar éste saca el de baja de contrato
3.- Añadir campo informativo tipo fecha (tanto en Geo como en HECEN) que sea "Fecha comunicación CCAA".

**Comment by JMunoz17** [comment]
Pregunta Belén,

Del punto 1, esto era solo para Geo verdad? Si me puedes indicar el mail a configurar para dejarlo puesto según lo suba.

Del punto 2, cual es el formato correcto para el mail de resolución de contrato?

No es este?

![image](https://user-images.githubusercontent.com/26869276/139893738-48018b84-f5bd-4b83-9b5b-adac37fdb8ff.png)

**Comment by bpesquero** [comment]
Jaime, toda la incidencia se refiere a Geo. Por tanto, sí, el punto 1 es sólo para Geo.
Punto2.- Ese texto es para clientes de más de 10 kW, para clientes de menos sería 

Estimado cliente:
 
Mediante la presente, se le requiere el pago de las cantidades adeudadas en concepto de consumo de energía eléctrica. A día de hoy, no ha satisfecho el pago de la/s factura/s emitida/s con fecha/s dd/ mm/aaaa, que asciende a un importe de xxx euros. De no abonarse la cantidad adeudada, a partir del día [fecha], la empresa distribuidora podrá suspender su suministro de electricidad.
Sin perjuicio de lo anterior, si usted cumple los requisitos para ser consumidor vulnerable, puede solicitar a una de las empresas comercializadoras de referencia acogerse al bono social, que supone un descuento sobre el precio voluntario para el pequeño consumidor (PVPC). El cambio de modalidad en el contrato para pasar a PVPC, siempre que no se modifiquen los parámetros recogidos en el contrato de acceso de terceros a la red, se llevará a cabo sin ningún tipo de penalización ni coste adicional. Siempre que se hayan acreditado los requisitos para ser consumidor vulnerable, una vez acogido al PVPC, el plazo para que su suministro de electricidad pueda ser suspendido, de no haber sido abonada la cantidad adeudada, se ampliará en 2 meses desde la fecha señalada anteriormente. El enlace a la página web de la CNMC donde encontrará los datos necesarios para contactar con la comercializadora de referencia es el siguiente: https://www.cnmc.es/ambitos-de-actuacion/energia/mercado-electrico#listados Adicionalmente, si usted cumple los requisitos para ser vulnerable severo, puede ponerse en contacto con los servicios sociales del municipio y comunidad autónoma donde reside, para que le informen sobre la posibilidad de atender el pago de su suministro de electricidad. Los requisitos para ser consumidor vulnerable vienen recogidos en el artículo 3 del Real Decreto 897/2017, de 6 de octubre, por el que se regula la figura del consumidor vulnerable, el bono social y otras medidas de protección para los consumidores domésticos de energía eléctrica, del que le remitimos un extracto.
a) Que su renta o, caso de formar parte de una unidad familiar, la renta conjunta anual de la unidad familiar a que pertenezca sea igual o inferior: – a 1,5 veces el Indicador Público de Renta de Efectos Múltiples (IPREM) de 14 pagas, en el caso de que no forme parte de una unidad familiar o no haya ningún menor en la unidad familiar; – a 2 veces el índice IPREM de 14 pagas, en el caso de que haya un menor en la unidad familiar; – a 2,5 veces el índice IPREM de 14 pagas, en el caso de que haya dos menores en la unidad familiar.
A estos efectos, se considera unidad familiar a la constituida conforme a lo dispuesto en la Ley 35/2006, de 28 de noviembre, del Impuesto sobre la Renta de las Personas Físicas y de modificación parcial de las leyes de los Impuestos sobre Sociedades, sobre la Renta de no Residentes y sobre el Patrimonio.
b) Estar en posesión del título de familia numerosa.
c) Que el propio consumidor y, en el caso de formar parte de una unidad familiar, todos los miembros de la misma que tengan ingresos, sean pensionistas del Sistema de la Seguridad Social por jubilación o incapacidad permanente, percibiendo la cuantía mínima vigente en cada momento para dichas clases de pensión, y no perciban otros ingresos.
   
Saludos,
[nombre de la empresa]

**Comment by JMunoz17** [comment]
Vale, pero a ver, por tenerlo claro.

El problema no es este:
_2.- Formato del email de resolución de contratos no es correcto, en vez de enviar éste saca el de baja de contrato_

Si no que el problema es que no estamos discriminando entre clientes con mas o menos de 10kw, no?

Eso en el caso de la resolución de contratos lo tenemos implementado en HECEN, pero no en GEO. No sé si quieres que lo incluyamos o lo dejemos para cuando rehagamos toda la gestión de impagos de Geo (creo que no tenemos ni el formato implementado). En caso de que la respuesta sea un sí, puedes confirmarme que es el mismo formato que para Hecen cambiando logos y enlaces..

**Comment by bpesquero** [comment]
Efectivamente, me informaron mal desde administración y no lo comprobé, el problema es que no se discrimina entre clientes de más de 10 y de menos de 20, al menos en esta comunicación. Me puedes confirmar si lo hacemos en el caso de reclamación formal de deuda?
Gracias

**Comment by JMunoz17** [comment]
Si, para la reclamación formal de deuda si que se hace.
Confírmame si lo hacemos para la resolución de contrato.

**Comment by bpesquero** [comment]
Si, habría que hacerlo también

**Comment by bpesquero** [comment]
La cuenta de correo a utilizar para recibir los emails de reclamación formal de deuda y resolución de contrato en sustitución de la del cliente es 
reclamaciondeuda@geoatlanter.com

**Comment by JMunoz17** [comment]
Subido a PRE y PROD.

**Comment by bpesquero** [comment]
Jiame, dijimos que el email al que se envían las notificaciones serían configurable desde "información de empresa". ES así? no lo veo.

**Comment by JMunoz17** [comment]
Así es Belén.

"Email propio - Impagos", abajo del todo en la pestaña de Comunicación.

![image](https://user-images.githubusercontent.com/26869276/143541383-f3137838-2a0e-4e08-b45f-6bdb51db9ac4.png)

**Comment by bpesquero** [comment]
Jaime, estamos viendo que el email de reclamación de deuda se está enviando a nuestro email, no se está enviando al cliente?
Esto es muy importante! Este email había que seguir enviándolo al cliente.

**Comment by JMunoz17** [comment]
Modificado y subido a PROD.
El email que se envía en GEO al importar las devoluciones se enviará al cliente directamente, los otros mails (los que se envían desde la pantalla de gestión de impagos) se enviarán a muestro email interno.
